### ISSUE TITLE
Replace "Download EAD" button's icon

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/vt-arclight#175]
I'm not sure if the current "plus" icon:

<img width="466" alt="Screen Shot 2022-11-07 at 5 00 50 PM" src="https://user-images.githubusercontent.com/101482/200441675-495debd2-6bdd-4f78-8a59-13122ed432b5.png">

is just a placeholder because we expect to have PDF download as well at some point (in which case we won't display an icon on the button), but in case we don't add another download option I'd suggest we replace the current plus icon with this one:

https://icons.getbootstrap.com/icons/download/

And add `0.25rem` right margin to the icon so there's a bit of space between it and the button label:

<img width="466" alt="Screen Shot 2022-11-07 at 5 00 18 PM" src="https://user-images.githubusercontent.com/101482/200441636-080110c5-b845-448c-8b9e-bf8cb8ecb24c.png">

**Comment by thatbudakguy** [comment]
what do you think about making this change in core, and also replacing the "gear" icon (which I don't think has much meaning here) with the "download" icon, in the case where there are multiple downloads?

(the labels in core are a bit confusing, "finding aid" here specifically means PDF, apparently)

![Image](https://user-images.githubusercontent.com/4924494/200692365-b00965c5-48be-46ce-86a4-ae73e4d35249.png)

**Comment by marlo-longley** [comment]
Hey @thatbudakguy in case it helps thinking about labels, I think likely most people would use the generated Finding Aid PDF you can get from ASpace or similar, so the term could be helpful

![Screen Shot 2022-11-08 at 17 54 27](https://user-images.githubusercontent.com/1328900/200693859-5bae0b8d-721c-4593-a103-3e96e438d4ad.png)

**Comment by marlo-longley** [comment]
Actually now that I generated this, I wonder if we should commit it to the Repo and offer it.

**Comment by ggeisler** [comment]
@thatbudakguy @marlo-longley 

Yes, I think updating the icons in core is the right thing. I didn't see the plus icon in core so that's why I posted the ticket here. And I was unsure whether there was some good reason we hadn't updated the gear icons in the pulldown, but I was thinking about writing that up so we should definitely do it.

I do think that "Finding aid" was a conscious word choice we made last workcycle (probably for reasons related to the points @marlo-longley makes).
